tumbleweed_known_changes.yml

```diff
@@ -1465,6 +1465,7 @@
 - files:
     - /usr/etc/keys
     - /usr/etc/keys/ima
+    - /usr/etc/keys/evm
```

Comment:
btw I see this as a kind of issue with script. It should be recursive by default. So everything under `/usr/etc/keys` should be ignored.